packages/wallet/src/redux/indirect-funding/player-a/actions.ts

```diff
@@ -0,0 +1,26 @@
+import { WalletProcedure } from '../../types';
```

Comment:
What are your thoughts on file and directory naming conventions? At the moment, we are snake casing file and directory names: `channelState`, `directFunding`, etc

Reply:
I think you mean camel casing. I guess I prefer camel casing, but was matching the directory names from this PR: https://github.com/magmo/apps/pull/285

I think I'll change them here, and request changes in #285

Reply:
👍